lesson3.slide

```diff
@@ -283,8 +283,8 @@ Outline:
 - *dynamic* *type*: type of assigned _value_
 - *dynamic* *value*: the actual value assigned
 - where's the problem?
-- interface type value is `nil` iff both dynamic value *and* dynamic type are `nil`
-- sometimes we get the impression that `nil != nil`
+- interface type value is `nil` if both dynamic value *and* dynamic type are `nil`
```

Comment:
`iff` is not a typo there, it means `if and only if`
https://en.wikipedia.org/wiki/If_and_only_if

Reply:
Sorry, @tisnik didn't know about this meaning in English